Shared/ContentView.swift

```diff
@@ -8,14 +8,22 @@
 import SwiftUI
 
 struct ContentView: View {
+    @EnvironmentObject var user: UserViewModel
+    
     var body: some View {
-        Text("Hello, Fun Gen!")
-            .padding()
+        NavigationView {
+            if user.userIsAuthenticatedAndSynced {
+                ProfileView()
+            } else {
+                AuthenticationView()
+            }
+        }
     }
 }
 
-struct ContentView_Previews: PreviewProvider {
-    static var previews: some View {
-        ContentView()
-    }
-}
+ struct ContentView_Previews: PreviewProvider {
+     static var previews: some View {
+         ContentView()
+        .environmentObject(UserViewModel())
```

Comment:
Not sure why SwiftLint didn't pick this up, but please indent all the modifiers (including this and the other ones, e.g. in profile view preview)

```suggestion
            .environmentObject(UserViewModel())
```

Reply:
Fixed